pkg/controller/provider/web/base/auth.go

```diff
@@ -59,7 +59,11 @@ func (r *Auth) Permit(ctx *gin.Context, p *api.Provider) (status int, err error)
 			return
 		}
 	}
-	allowed, err := r.permit(token, p)
+	if p.ObjectMeta.UID != "" {
```

Comment:
don't you mean
```suggestion
	if p.ObjectMeta.UID == "" {
```

Reply:
wow, thanks, fixing :+1:

Reply:
fixed, 
( now we don't need to fix in the test, but it's a good fix, leaving :-) )